frontend/src/utils/api.ts

```diff
@@ -407,19 +409,30 @@ function apiFetch<R extends ResponseWithData<any>>(method: HTTPMethod, route: st
                         undefined,
         })
 
-        return request.then(res => {
-            if (options.abort) {
-                delete ongoingRequests[baseRoute]
-            }
-            return res
-        })
-            .then((response) => attachData<R>(response))
-            .then(response => {
-                if (response.ok) {
-                    return response;
-                }
-                return Promise.reject(createAPIError(response));
-            })
+        return request
+          .then(res => {
+              if (options.abort) {
+                  delete ongoingRequests[baseRoute]
+              }
+              return res
+          })
+          .then((response) => attachData<R>(response))
+          .then(response => {
+              if (response.ok) {
+                  return response;
+              }
+              if (options.notifyError) {
+                  let detail = response.data.detail
+                  if (Array.isArray(detail)) {
+                    detail = detail.join('. ')
```

Comment:
the ". " separator is a weird choice. Most error messages should have a dot already at the end of the sentence... I would use "; " instead to separate them.

Reply:
DONE